CryptoTechReminderSystem.Test/UseCase/GetLateDevelopersTests.cs

```diff
@@ -88,30 +101,98 @@ public void CanGetHarvestDevelopers()
         public void CanGetSlackDevelopers()
         {
             var harvestGatewaySpy = new HarvestGatewaySpy();
-            var slackGatewaySpy = new SlackGatewaySpy();            
-            var getDevelopers = new GetLateDevelopers(slackGatewaySpy, harvestGatewaySpy, harvestGatewaySpy);
+            var slackGatewaySpy = new SlackGatewaySpy();
+            var clock = new ClockStub(
+                new DateTimeOffset(
+                    new DateTime(2019, 03, 01, 10, 30, 0)
+                )
+            );
```

Comment:
extract to a one time setup here

Reply:
Resolved in https://github.com/chubberlisk/crypto_tech/pull/23/commits/6be324ccbeffbc71f70eebe10457548fb15896dc.